src/main/haskell/language-kore/test/parser/Data/Kore/ASTUtils/ASTUtilsTest.hs

```diff
@@ -0,0 +1,217 @@
+{-# LANGUAGE MultiParamTypeClasses  #-}
```

Comment:
I didn't get an answer to one of my previous questions, so I'll try asking it this way: What's the test coverage for the new code?

Reply:
Do you mean `hpc report` ? About 70-80%

Reply:
We usually use `stack test --coverage` and look at the html report, I guess that may use `hpc` behind the scenes.

That number, by itself, does not mean much. We are aiming for 100% coverage. Until now we allowed the coverage to be lower for things like automatic instance derivations and struct getters, but we usually want close to 100% for everything else. So, if the missing 20-30% are things like that, it should be fine.